data/korath/korath variants.txt

```diff
@@ -678,6 +682,12 @@ ship "Ra'gru Ik 618" "Ra'gru Ik 618 (Grab)"
 	turret "Banisher Grav-Turret"
 
 
+# This variant needs to have the same load-out as the default 618 to
+# keep the intended difficulty level of the Korath Artifacts 2  mission
+ship "Ra'gru Ik 618" "Ra'gru Ik 618 (Ground Assault)"
+	"crew" 794
```

Comment:
So, the Ground Assault variant is merely a regular 618 with a higher crew count?

Given the previous comments, this leads me to suspect that there is a concern that in the future someone might modify this variant to actually be outfitted differently for a ground assault role. 

If this is indeed the case, and there's no actual reason for why the Korath are using a ground assault ship here other than to merely have a high crew count to make it hard to capture this ship; then I would suggest using a more specific "Ra'gru Ik 618 (Sentry)" name, thus allowing the (Ground Assault) label to be used at some point in the future when it is, indeed, actually being outfitted as part of a Ground Assault operation.

```suggestion
ship "Ra'gru Ik 618" "Ra'gru Ik 618 (Sentry)"
	"crew" 794
```